_includes/post-meta.html

```diff
@@ -1,19 +1,31 @@
 {% if page.author %}
-  {% assign author = site.authors[page.author] %}
+{% assign author = site.authors[page.author] %}
 {% else %}
-  {% assign author = site.author %}
+{% assign author = site.author %}
 {% endif %}
 
 {% if author.github %}
-    <!-- {% avatar {{ author.github }} size=32 %} -->
+<!-- {% avatar {{ author.github }} size=32 %} -->
 {% endif %}
 
 {% if author.name %}
-    <!-- Written by <span itemprop="author" itemscope itemtype="http://schema.org/Person"><span itemprop="name">{{ author.name }}</span></span><br> -->
+<!-- Written by <span itemprop="author" itemscope itemtype="http://schema.org/Person"><span itemprop="name">{{ author.name }}</span></span><br> -->
 {% endif %}
 
-<time datetime="{{ page.date | date_to_xmlschema }}" itemprop="datePublished">
-    {% assign date_format = site.date_format | default: "%b %-d, %Y" %}
-    {{ page.date | date: date_format }}
-    {% include editme.html %}  
+<time datetime="{{ page.last_modified_at | date_to_xmlschema }}" itemprop="datePublished">
+  {{ page.last_modified_at | date: "%b %-d, %Y" }}
+  {% include editme.html %}
 </time>
+<div>
+  {% assign three_months_ago = 'now' | date: '%s' | minus: 15552000 %}
+  {% assign page_date = page.last_modified_at | date: '%s' %}
+  {% assign diff_date = three_months_ago | minus: page_date %}
+
+  {% if diff_date >= 0 %}
+  <div class=uk-alert-warning uk-card" style="margin-top: 25px !important;" uk-alert>
+    <b>Note:</b>
+    This page has been published more than 6 months ago, and parts
+    of its contents are very likely to be obsolete by now.
```

Comment:
This language is too strong - "Very likely" is an overreach. Most likely, a majority of the 6+ month old pages are still up to date, and there are more recently updated pages (zero to dapp) that are "outdated".

Maybe an option is to increase this threshold to something like 1 year?

I think there would be more value in manually adding a notice like this to specific pages that we know we need to update. And then, we should be tracking those updates somewhere.

Reply:
We could add them manually as well, this would be another workstream that requires us to go through each page and review it. Is that something we want to sign up for right now? I couldn't judge by myself and I assume it requires multiple people going through pages.

We might want to use both approaches, actually...
- make the callout for _really_ old docs, 1 year+ (and soften the language as per feedback above)
- identify only the most relevant pages for review and add the callout manually to these

Reply:
If we're going to automate this, I would recommend softening the language as well to something like "may be partially out of date" (since if a page is _entirely_ out of date, we're likely to have removed it).

Reply:
I hear you both, the language needs to change